clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -783,6 +783,7 @@ private JoinGroupRequest createJoinGroupRequest(int version) {
                             .setGroupId("group1")
                             .setSessionTimeoutMs(30000)
                             .setMemberId("consumer1")
+                            .setGroupInstanceId(JoinGroupRequest.EMPTY_GROUP_INSTANCE_ID)
```

Comment:
Should we still need these calls if we can get rid of const `EMPTY_GROUP_INSTANCE_ID` with optional?

Reply:
We don't